tests/test_presentation.py

```diff
@@ -573,3 +573,104 @@ def test_agent_type(self):
         element = agent_xml.agentType
         assert element == agent.agent_type
         assert has_premis_namespace(element)
+
+
+class TestDoSimpleXMLAssignment:
+
+    @pytest.fixture
+    def record_obj(self):
+        class Record(object):
+            value = None
+        return Record()
+
+    def test_attribute_not_set_when_element_value_is_none(self, record_obj):
```

Comment:
Could we rename this test to indicate the root (fieldname not matching element name) of the test?